config/cloudformation.yaml

```diff
@@ -65,6 +65,12 @@ Parameters:
   DomainName:
     Type: String
     Description: The DomainName of the ALB, should contain the trailing dot stuff.zone.example.com.
+  EditionsPreviewLambdaSecurityGroupId:
+    Type: String
+    Description: A security group id for editions preview lambda to access AR
+  EditionsPublishedLambdaSecurityGroupId:
+    Type: String
+    Description: A security group id for editions published lambda to access AR    
```

Comment:
My only suggestion would be to include the word 'backend' in the name or description as there are a lot of different editions lambdas - 'editions backend published lambda'